src/gt4py/frontend/gtscript_frontend.py

```diff
@@ -423,7 +425,7 @@ def visit_Call(self, node: ast.Call, *, target_node=None):
     def visit_Expr(self, node: ast.Expr):
         """ignore pure string statements in callee"""
         if not isinstance(node.value, ast.Str):
-            return super().visit_Expr(node)
+            return super().visit(node)
```

Comment:
Unrelated fix that my editor's linter complained about.

Reply:
Yes, the proper way to visit a node is calling the general visit method dispatcher